backend/src/db-utils.js

```diff
@@ -376,6 +400,27 @@ const queryActiveAccountsList = async () => {
   );
 };
 
+const queryDepositAmountAggregatedByDate = async () => {
+  return await queryRows(
+    [
+      `SELECT 
+          TIMESTAMP 'epoch' + DIV(DIV(receipts.included_in_block_timestamp, 1000000000), 60 * 60 * 24) * INTERVAL '1 day' AS "date",	
+          CAST(SUM((action_receipt_actions.args->>'deposit')::numeric) AS TEXT) AS total_deposit_amount
+        FROM action_receipt_actions
+        JOIN execution_outcomes ON execution_outcomes.receipt_id = action_receipt_actions.receipt_id
+        JOIN receipts ON receipts.receipt_id = action_receipt_actions.receipt_id
+        WHERE action_receipt_actions.action_kind IN ('FUNCTION_CALL', 'TRANSFER')
```

Comment:
We should filter out `receipts.predecessor_account_id = 'system'`, so we don't account for transaction refunds.

Reply:
@frol 
what is the transaction for predecessor accountId is `system`??? I do not clear that